src/main/java/org/jenkinsci/plugins/sma/SMARunner.java

```diff
@@ -58,16 +57,17 @@ public SMARunner(EnvVars jobVariables, String prTargetBranch, SMAJenkinsCIOrgSet
         }
         // Configure using pull request logic
         if (!prTargetBranch.isEmpty() && !shaOverride) {
-            deployAll = false;
-            git = new SMAGit(pathToWorkspace, currentCommit, prTargetBranch, SMAGit.Mode.PRB);
-            previousCommit = git.getPrevCommit();
+//            deployAll = false;
```

Comment:
No sure why deployAll is set to false here. I think it should always deploy all in case for some reason deployAll is set to "true" imho. Or is this implemented for a special reason?